tbx/static_src/sass/components/_rich-text.scss

```diff
@@ -1,11 +1,42 @@
 @use "config" as *;
 
 .rich-text {
+    @include font-size(body);
+
+    h3 {
+        @include font-size(heading-three);
+        margin-bottom: $spacer-small;
+    }
+
+    h4 {
+        @include font-size(heading-four);
+        margin-bottom: $spacer-small;
+    }
+
     ul {
         list-style: inside disc;
+        margin-bottom: $spacer-small;
     }
 
     ol {
         list-style: inside decimal;
+        margin-bottom: $spacer-small;
+    }
+
+    a {
```

Comment:
The ticket mentions to apply link styles with a class but we'll need to target the `<a>` directly in rich text blocks.